src/iface/ethernet.rs

```diff
@@ -1,6 +1,7 @@
 // Heads up! Before working on this file you should read the parts
 // of RFC 1122 that discuss Ethernet, ARP and IP.
 
+use core::cmp::min;
```

Comment:
`use core::cmp` and then `cmp::min(...)`.

Reply:
Updated.